src/cartridges/app_adyen_SFRA/cartridge/client/default/js/paypalExpress.js

```diff
@@ -0,0 +1,225 @@
+const {
+  getPaymentMethods,
+  updateLoadedExpressMethods,
+  checkIfExpressMethodsAreReady,
+} = require('./commons');
+const helpers = require('./adyen_checkout/helpers');
+const { PAYPAL } = require('./constants');
+
+async function callPaymentFromComponent(data, component) {
+  try {
+    $.spinner().start();
+    const response = await fetch(window.makeExpressPaymentsCall, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+    const { action, errorMessage = '' } = await response.json();
+    if (response.ok && action) {
+      component.handleAction(action);
+    } else {
+      throw new Error(errorMessage);
+    }
+  } catch (e) {
+    component.handleError();
+  }
+}
+
+async function saveShopperDetails(details, actions) {
+  return $.ajax({
+    url: window.saveShopperData,
+    type: 'post',
+    data: {
+      shopperDetails: JSON.stringify(details),
+    },
+    success() {
+      actions.resolve();
+    },
+    error() {
+      $.spinner().stop();
+    },
+  });
+}
+
+function redirectToReviewPage(data) {
+  const redirect = $('<form>').appendTo(document.body).attr({
+    method: 'POST',
+    action: window.checkoutReview,
+  });
+  $('<input>')
+    .appendTo(redirect)
+    .attr({
+      name: 'data',
+      value: JSON.stringify(data),
+    });
+
+  redirect.submit();
+}
+
+function makeExpressPaymentDetailsCall(data) {
+  return $.ajax({
+    type: 'POST',
+    url: window.makeExpressPaymentDetailsCall,
+    data: JSON.stringify({ data }),
+    contentType: 'application/json; charset=utf-8',
+    async: false,
```

Comment:
It seems this is deprecated. Are we sure we want to use async:false here? [Link](https://stackoverflow.com/questions/11448011/jquery-ajax-methods-async-option-deprecated-what-now) for more info.

Reply:
You're right, it is deprecated, but there is not any other official alternative provided by jquery. There exists only manual solutions. 
We have discussed this as part of unifying all the calls we do, however we need to come up with a generic alternative as we use it in quite some calls in the cartridge.